### DISCUSSION TITLE
Add react-hook-form to React Native Directory

### DISCUSSION THREAD

**Discussion by irekrog** [react-hook-form/react-hook-form#3827] [Ideas] [answered]
Hi @bluebill1049 

What do you think about adding `react-hook-form` to React Native Directory?

https://reactnative.directory/
https://github.com/react-native-community/directory#how-do-i-add-a-library

**Comment by bluebill1049** [comment, marked as answer]
100% would be able to do that?

**Reply by irekrog** [reply]
Yeah, so I have just created PR :)

**Reply by bluebill1049** [reply]
awesome! thank you very much @irekrog ❤️

**Reply by irekrog** [reply]
https://reactnative.directory/?search=react-hook-form 🎉

**Reply by bluebill1049** [reply]
awesome! thank you.
